.github/workflows/prereleases.yml

```diff
@@ -0,0 +1,48 @@
+# name: Prerelease
```

Comment:
why don't we keep this around? looks like it'll still work, sans changelogs, which is fine.

Reply:
I can not really have only release or prerelease run based on the current Event trigger which is `push to main`